src/tensors/cpu/tensor_operators.cpp

```diff
@@ -136,15 +138,14 @@ void Split1(std::vector<Tensor>& outputs, const Tensor in) {
 
     // set last parameter to 1 to enable += instead of =
     // @TODO: do this in a more principled ways accross all/most kernels
-    cpu::gInsertCols(out->data(),
-                     in->data(),
-                     rows,
-                     cols_out,
-                     cols_out,
-                     cols_in,
-                     0,
-                     offset,
-                     1);
+    cpu::gInsertCols<true>(out->data(),
```

Comment:
consider adding a comment, e.g. `</*add=*/true>` for better readability (also consider adding such fields for the non-obvious 0 argument here and above)